apps/workflow_editor/views/App.js

```diff
@@ -6,35 +6,24 @@ import React, { Component } from 'react';
 
 import AppContainer from 'common-web-ui/views/AppContainer';
 
+import { navigation } from '../config/routes';
+
 export default class App extends Component {
 
   state = {
-    title: this.title
+    navigation
   };
 
   render() {
+    document.body.style.overflow = 'hidden';
```

Comment:
Expected ')' to match '{' from line 17 and instead saw '.'.<br>'function closure expressions' is only available in Mozilla JavaScript extensions (use moz option).<br>Unexpected ';'.